policies/committers.html

```diff
@@ -38,49 +38,46 @@ <h2>How to become a committer?</h2>
               community and its
               <a href="/community/conduct.html">code of conduct</a>.</p>
 
-              <p>To become a committer, start by contributing code. Read our
+              <p>To become a committer, start by contributing code (GitHub pull requests).
+              Read our
```

Comment:
I don't think think that this addition is useful, it just distracts from the clear and concise statement of the sentence. (IOW I would revert the change.)

A useful description of _how_ the code is contributed would require a lot more words, but that is outside the scope of this policy document. And just adding a laconic "(GitHub pull requests)" does not really explain anything.

Reply:
I reverted this change since it is entirely tangential to the main point of the PR anyway.